packages/mobile/ios/AudiusReactNative.xcodeproj/project.pbxproj

```diff
@@ -453,7 +453,6 @@
 				CODE_SIGN_ENTITLEMENTS = "AudiusReactNative/Audius Music.entitlements";
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_TEAM = LRFCG93S85;
-				EXCLUDED_ARCHS = "";
```

Comment:
Are we sure we want to remove these? @dylanjeffers already removed the arm64 exclusion